<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommonDialog</class>
 <widget class="QDialog" name="CommonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="dialogHLayout">
   <item>
    <widget class="QWidget" name="centralWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>680</width>
       <height>350</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>680</width>
       <height>350</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="title" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>36</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="titleHLayout">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>146</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="titleName">
           <property name="text">
            <string>System Prompt</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>146</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="closeButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#closeButton{border-image: url(:/images/res/images/ic_close_tab.png);}
QPushButton#closeButton:pressed{border-image: url(:/images/res/images/ic_close_tab_p.png);}</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="text">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="content" native="true"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttonHLayout">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>40</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="cancelButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>110</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
color: rgb(255, 255, 255);
border-radius:14px 14px 14px 14px;
border: 1px solid #FFFFFF;
}
QPushButton:hover{
background-color: rgb(85, 255, 255);
}

QPushButton:pressed{
background-color: rgb(144, 144, 144);
}</string>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="okButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>110</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
color: rgb(255, 255, 255);
border-radius:14px 14px 14px 14px;
background-color: rgb(0, 165, 255);
}
QPushButton:hover{
background-color: rgb(85, 255, 255);
}

QPushButton:pressed{
background-color: rgb(144, 144, 144);
}</string>
          </property>
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>40</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="save">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>110</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
color: rgb(255, 255, 255);
border-radius:14px 14px 14px 14px;
background-color: rgb(0, 165, 255);
}
QPushButton:hover{
background-color: rgb(85, 255, 255);
}

QPushButton:pressed{
background-color: rgb(144, 144, 144);
}</string>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="abandon">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>110</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
color: rgb(255, 255, 255);
border-radius:14px 14px 14px 14px;
background-color: rgb(0, 165, 255);
}
QPushButton:hover{
background-color: rgb(85, 255, 255);
}

QPushButton:pressed{
background-color: rgb(144, 144, 144);
}</string>
          </property>
          <property name="text">
           <string>Don't Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelButton_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>110</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
color: rgb(255, 255, 255);
border-radius:14px 14px 14px 14px;
border: 1px solid #FFFFFF;
}
QPushButton:hover{
background-color: rgb(85, 255, 255);
}

QPushButton:pressed{
background-color: rgb(144, 144, 144);
}</string>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
